tmt/steps/discover/fmf.py

```diff
@@ -157,6 +158,13 @@ class DiscoverFmfStepData(tmt.steps.discover.DiscoverStepData):
         show_default=True,
         help="Copy only immediate directories of executed tests and their required files.")
 
+    # Edit discovered tests
+    adjust_tests: Optional[list[_RawAdjustRule]] = field(
+        default_factory=list,
+        normalize=lambda key_address, raw_value, logger: [] if raw_value is None
+        else ([raw_value] if not isinstance(raw_value, list) else raw_value),
```

Comment:
I'd call it `normalize_adjust()` - I'm not sure we want to normalize any list of dictionaries, this is a pretty special case because we consciously decided not to even inspect content of those dictionaries, which is not the case of other dictionaries tmt reads (raw fmf id, raw phases, etc.)

Reply:
ack